### PR TITLE
Added sample code to multiply 2 numbers along with unit tests

### LINKED ISSUE #1: Add a sample code to multiply 2 numbers along with unit tests.

### PR DESCRIPTION
**Pull request by aikido-sensei**
Closes #1

Added sample code to multiply 2 numbers along with unit tests

### CODE REVIEW

File: unit_tests/multiply_two_numbers.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+def multiply_2_nums(num1: int, num2: int) -> int:
+    """
+
```
Comment (silvs110) [inline unit_tests/multiply_two_numbers.py]: Method's description is missing

File: unit_tests/multiply_two_numbers.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    :param num1: int : Number 1 used in the multiplication
+    :param num2: int: Number 2 used in the multiplication
+    :return: int: The result of the multiplication of num1 and num2
+    """
```
Comment (the-c0d3br34k3r) [inline unit_tests/multiply_two_numbers.py:7]:
```suggestion
def multiply_2_nums(num1: int, num2: int) -> int:
    """

    :param num1: int : Number 1 used in the multiplication
    :param num2: int: Number 2 used in the multiplication
    :return: int: The result of the multiplication of num1 and num2
    """
```
```suggestion
def multiply_two_nums(num1: int, num2: int) -> int:
    """
    Multiply two numbers and return the result.
    :param num1: int : Number 1 used in the multiplication
    :param num2: int: Number 2 used in the multiplication
    :return: int: The result of the multiplication of num1 and num2
    """
```

Comment (the-c0d3br34k3r) [inline unit_tests/multiply_two_numbers.py]:
```suggestion
def multiply_two_nums(num1: int, num2: int) -> int:
    """
    Multiply two numbers and return the result.
    :param num1: int : Number 1 used in the multiplication
    :param num2: int: Number 2 used in the multiplication
    :return: int: The result of the multiplication of num1 and num2
    """
```
```suggestion
def multiply_2_nums(num1: int, num2: int) -> int:
    """

    :param num1: int : Number 1 used in the multiplication
    :param num2: int: Number 2 used in the multiplication
    :return: int: The result of the multiplication of num1 and num2
    """
```

File: unit_tests/multiply_two_numbers_test.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        """
+        [-2,-2] : 4
+        [2,-2] : -4
+        [2,2] : 4
+        [3, 0] : 0
+        [2, "iha"] : AttributeError
+        [2, 2.5] : AttributeError
+        [None, 3] : AttributeError
+
+        """
```
Comment (the-c0d3br34k3r) [inline unit_tests/multiply_two_numbers_test.py]:
```suggestion
        """
        Tests the multiply_two_numbers() method for the following input combinations.
        [-2,-2] : 4
        [2,-2] : -4
        [2,2] : 4
        [3, 0] : 0
        [2, "iha"] : AttributeError
        [2, 2.5] : AttributeError
        [None, 3] : AttributeError

        """
```
```suggestion
        """
        [-2,-2] : 4
        [2,-2] : -4
        [2,2] : 4
        [3, 0] : 0
        [2, "iha"] : AttributeError
        [2, 2.5] : AttributeError
        [None, 3] : AttributeError

        """
```

**Comment by the-c0d3br34k3r** [comment]
Please refer the issue number in the commit message.

**Review by the-c0d3br34k3r — CHANGES_REQUESTED** [review]
Reviewed. Requested changes.

File: unit_tests/multiply_two_numbers_test.py [hunk 4]
```diff
@@ -0,0 +1,32 @@
+import unittest
+from unit_tests.multiply_two_numbers import multiply_2_nums
+
+
+class MultiplyTwoNumsTest(unittest.TestCase):
+
+    def test_multiplication(self):
```
Comment (the-c0d3br34k3r) [inline unit_tests/multiply_two_numbers_test.py]:
```suggestion
    def test_multiplication(self) -> None:
```

File: unit_tests/multiply_two_numbers.py [hunk 5]
```diff
@@ -0,0 +1,14 @@
+def multiply_two_nums(num1: int, num2: int) -> int:
+    """
+    Multiply two numbers and return the result.
+    :param num1: int : Number 1 used in the multiplication
+    :param num2: int: Number 2 used in the multiplication
+    :return: int: The result of the multiplication of num1 and num2
```
Comment (the-c0d3br34k3r) [inline unit_tests/multiply_two_numbers.py]: Please refer the "reStructured text" example and conform to it.
https://realpython.com/documenting-python-code/#restructured-text-example

**Review by the-c0d3br34k3r — CHANGES_REQUESTED** [review]
Requested changes. Please squat all commits as well.

**Comment by the-c0d3br34k3r** [comment]
Please refer issue in commit message.

File: unit_tests/multiply_two_numbers_test.py [hunk 6]
```diff
@@ -0,0 +1,31 @@
+import unittest
+from unit_tests.multiply_two_numbers import multiply_two_nums
+
+
+class MultiplyTwoNumsTest(unittest.TestCase):
+
+    def test_multiplication(self):
```
Comment (the-c0d3br34k3r) [inline unit_tests/multiply_two_numbers_test.py]:
```suggestion
    def test_multiplication(self) -> None:
```

File: unit_tests/multiply_two_numbers_test.py [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+        [-2,-2] : 4
+        [2,-2] : -4
+        [2,2] : 4
+        [3, 0] : 0
+        [2, "iha"] : AttributeError
+        [2, 2.5] : AttributeError
+        [None, 3] : AttributeError
+        """
+
+        test_inputs = {
```
Comment (the-c0d3br34k3r) [inline unit_tests/multiply_two_numbers_test.py]:
```suggestion
        test_inputs: dict = {
```

**Review by the-c0d3br34k3r — CHANGES_REQUESTED** [review]
Changes have been requested.

**Comment by aikido-sensei** [comment]
> Please refer the issue number in the commit message.

Done!

**Comment by aikido-sensei** [comment]
> Reviewed. Requested changes.

Fixed

**Comment by aikido-sensei** [comment]
> Requested changes. Please squat all commits as well.

Done!

**Comment by aikido-sensei** [comment]
> Please refer issue in commit message.

Done!

**Comment by aikido-sensei** [comment]
> Changes have been requested.

Corrected!

**Review by the-c0d3br34k3r — CHANGES_REQUESTED** [review]
Please squash all the commits into a single commit.

**Comment by aikido-sensei** [comment]
> Please squash all the commits into a single commit.

Done!

**Review by the-c0d3br34k3r — APPROVED** [review]

**Review by silvs110 — APPROVED** [review]

File: unit_tests/multiply_two_numbers.py (hunk 1 above)
Comment (aikido-sensei) [reply unit_tests/multiply_two_numbers.py]: Done!

**Pull request merged**